lib/Driver/UnixToolChains.cpp

```diff
@@ -9,6 +9,8 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
+#include <iostream>
```

Comment:
If possible, please use the LLVM APIs available. We're already including them.

Reply:
@etcwilde, I have removed `#include <iostream>` as it was only being used for print some log messages while I was debugging.